src/screens/SearchScreen/components/SearchScreenContainer.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import SearchBar from './SearchBar';
+import DictionaryEntriesContainer from './DictionaryEntriesContainer';
+import translate from '../../../api/api';
+
+const BAD_SEARCH = 'No results. Did you spell the word correctly?';
+const SOURCE_LANG = 'en';
+const TARGET_LANG = 'fr';
+
+const updateData = (searchTerm, setData, setEmptyMessage) => {
+  translate(SOURCE_LANG, TARGET_LANG, searchTerm)
+    .then(responseData => {
+      responseData = responseData.data;
+      if (responseData === undefined) {
+        setData([]);
+        setEmptyMessage(BAD_SEARCH);
+      } else {
+        setData(responseData);
+        setEmptyMessage('');
+      }
+    })
+    .catch(error => {
+      setData([]);
+      setEmptyMessage(BAD_SEARCH);
+    });
+};
+
+const SearchScreenContainer = ({navigation}) => {
+  const [data, setData] = useState([]);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [emptyMessage, setEmptyMessage] = useState('');
```

Comment:
If you're interested, you could add a state variable to track whether or not a search is in progress (keep track of whether the screen is loading data or not). Then you could show some kind of indicator in the UI while that is happening to let users know the app is working and a result is coming.